Assets/_Scripts/3Managers/IslandManager.cs

```diff
@@ -36,10 +37,10 @@ public void TryPlaceBuilding(Tile tile, BuildingType buildingType)
         //TODO: This needs to be moved at a later point.
         var validator = buildingType.GetBuildingValidator(tile);
         var buildingErrors = validator.ValidateResources(buildingType);
-        if (buildingErrors.Any()) { buildingErrors.ForEach(x => Debug.Log(x)); return; }
+        if (buildingErrors.Any()) { buildingErrors.ForEach(x => Logger.LogMessage(x, Logger.LogType.Error)); return; }
```

Comment:
The logger has no good support for multiple error messages, showing them all in the same position:
![image](https://user-images.githubusercontent.com/16051709/168077978-1f571103-3d97-457f-a689-745191eaebc7.png)